### PR TITLE
Call for user stories

### PR DESCRIPTION
**Pull request by kjetilk**
This is a document that calls for user stories around the server.

### CODE REVIEW

File: server-user-stories-intro.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# User Stories for the Solid Server
+
+To enable more people to get involved, to improve code quality to
+production grade and to accelerate development, I would like to hear
```
Comment (TallTed) [inline server-user-stories-intro.md]: "grade and" should have a comma, as "grade, and"

File: server-user-stories-intro.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+your User Stories for the server.
+
+This is to better understand better what people want and need from the
+server. User Stories are short descriptions of a need written from the
+perspective of a user,
+[see e.g. this article](https://www.mountaingoatsoftware.com/agile/user-stories). They
```
Comment (TallTed) [inline server-user-stories-intro.md]: "see e.g. this article" would be better "see this article for an example"

File: server-user-stories-intro.md (hunk 1 above)
Comment (kjetilk) [reply server-user-stories-intro.md]: thanks!

File: server-user-stories-intro.md (hunk 2 above)
Comment (kjetilk) [reply server-user-stories-intro.md]: yeah, that's better!

**Comment by Mitzi-Laszlo** [comment]
Solid Chat User Stories: 

Essential: 
* send a text message to a contact
* create a group with select contacts with a specific group name 
* send a photo to a contact
* send a video to a contact 
* send an audio clip to a contact 
* call a contact 
* call a group of contacts

**Comment by kjetilk** [comment]
Oh, thanks, @Mitzi-Laszlo , but it is not supposed to go here, but in the wiki, and rather be on the specified form. Also, it is rather hard to tell what is relevant for the server, most of these stories are rather within what the server is already specified to do. I'm afraid these things are rather subtle.

**Comment by kjetilk** [comment]
@pmcb55 pointed out to me that he could edit the Wiki, it doesn't seem to be an easy to way to enable non-team write access to a Github wiki page. It seems it is possible to open it more generally, but it is somewhat unclear to me. 

The proposed Wiki page is pretty useless if the community at large can't comment, so I think the best we can do is to move it to another Wiki. Possibly the W3C one?

**Comment by Mitzi-Laszlo** [comment]
As a person I want to have a recognisable unique identity so that I can communicate to others who I am and what belongs to me 

As a person I want to have a place online to collect and store information about myself so that I can present myself to others online

As a person, I want to communicate a public facing identity so that others can learn about me  

As a person, I want to view the public facing identities of others to learn about them. 

As I person I want to use information about me so that I can get stuff done efficiently to my own benefit 

As a person I want to understand the implications of my data sharing preferences and choose a fixed data sharing preference that will operate on all applications until I choose to change those data sharing preferences. 

As a parent or guardian, I want to set up a recognizable unity identity for my child so that they can communicate to others who they are and what belongs to them

As a parent or guardian, I I want to have a place online to collect and store information about my child so that my child can present themselves to others online

As I parent or guardian, I want to use information about my child to get stuff done efficiently to the benefit of my child

As a parent, I want to communicate a public facing identity of my child so that others can learn about my child  
Solid Chat
As a person I want to send a text message to a contact so that I can communicate with other people one on one 

As a person I want to send a text message to a defined group of people so that I can communicate with other people in a group online   

As a person I want to send a photo to a contact so that I can communicate with other people or groups of people 

As a person I want to send a video to a contact so that I can communicate with other people or groups of people 

As a person I want to send an audio clip to a contact so that I can communicate with other people or groups of people 

As a person I want to call a person or group of people so that I can communicate with them. 
Solid Pay
As a person, I want to transfer money to another person or institution to pay for a product or service or tax.

**Comment by MattDorrian** [comment]
As a person I wish to trust the systems that store and collect my personal information so the internet and web can fulfill their natural potential.

**Comment by MattDorrian** [comment]
As a person I wish for my next of kin to recieve my chosen life history, important documents and associated monetary accounts in the event of my death.

**Comment by kjetilk** [comment]
Thanks, folks, but it wasn't the idea to submit it here, this was just about how we submit. Hold on a bit, @Mitzi-Laszlo and I found that we should have a repo for this.

**Comment by megoth** [comment]
Let's move all of the suggested user stories that's here to https://github.com/solid/user-stories ^_^

**Comment by kjetilk** [comment]
OK, so now, it is up at https://github.com/solid/user-stories 

So, please go there! 

@MattDorrian I particularly liked your story about life history, that's an important one, so I hope you can be bothered to resubmit.

**Pull request closed without merging**